vertical-pod-autoscaler/deploy/vpa-rbac.yaml

```diff
@@ -4,6 +4,18 @@ kind: ClusterRole
 metadata:
   name: system:metrics-reader
 rules:
+  - apiGroups:
+      - "custom.metrics.k8s.io"
+    resources:
+      - "*"
+    verbs:
+      - "*"
```

Comment:
This is too much access for a reader role. Only grant read access.

Reply:
Fixing.